src/v2/Apps/Artwork/Components/ArtworkMeta.tsx

```diff
@@ -111,6 +125,20 @@ export class ArtworkMeta extends Component<ArtworkMetaProps> {
     )
   }
 
+  renderZendeskScript() {
+    const { artwork } = this.props
+    if (
+      artwork.is_in_auction ||
+      !artwork.is_acquireable ||
+      !artwork.isOfferable ||
```

Comment:
Not sure why we had the 🐍 -case previously. 🙈

Reply:
I think this is a result of MP V1 🤷‍♀️